### PR TITLE
Make stable_days field in update form to use mandatory_days_in_testing

### PR DESCRIPTION
**Pull request by mattiaverga**
This will make the "stable days" field fo the new update form to display an "auto" placeholder when creating a new update, while when editing an existing update the minimum accepted value will be the mandatory_days_in_testing for that update.

It was quite confusing for me to have "Auto-request stable based on time" flagged and "Stable days" with an empty value (zero?). This should make things clearer.

Signed-off-by: Mattia Verga <mattia.verga@tiscali.it>

### CODE REVIEW

File: bodhi/server/templates/new_update.html
Comment (cverna) [inline bodhi/server/templates/new_update.html]: I think actually we want this to be 0 since we will have releases with 0 mandatory days in testing like rawhide.

**Review by cverna — COMMENTED** [review]
Thanks, looks good just one small comment

**Review by cverna — APPROVED** [review]
:+1:

**Comment by cverna** [comment]
@abompard I forgot to add the backport 4.1 label to this PR but I think it could go into that build, since it improve the UI for the autotime updates.
I leave it to you to decide if we want to include this one in the 4.1 or not

**Comment by abompard** [comment]
I wonder if Mergify can pick it up if I add the tag even after it's merged or if I should just cherry-pick it. Testing that.

**Comment by abompard** [comment]
It does! Way to go Mergify! :-)

**Pull request merged**
